TLM/TLM/State/OptionsTabs/GeneralTab_InterfaceGroup.cs

```diff
@@ -0,0 +1,172 @@
+namespace TrafficManager.State {
```

Comment:
General file naming comment. In the project you can drag files into other files to group them like in folders.
And if you do so, files can have `.` in their names instead of `_`.

Reply:
By the time I'm finished refactoring option groups the `...Tab` files will be practically empty. My intention was that I'd take the skeletal code from those files and put it in to the main `MakeSettings` method in `OptionsManager`, then delete the `Tab` files and finally simplify the names of the `...Group` files to remove the tab name.

I'm also thinking of adding more UI helpers for things like groups, tabs, etc. So options would register with their parent group, which registers with parent tab, which registers with parent tab strip.... then when locale changes we can just trigger something on tab strip which in turn will refresh label/tooltip of the UI hierarchy beneath it. It would also mean, for example, that we could disable UI components when options screen is not visible, so users who don't have FPS Booster aren't having TM:PE options eat up their CPU cycles.